lib/kintsugi/cli.rb

```diff
@@ -171,33 +160,21 @@ def uninstall_kintsugi_driver_globally
     end
```

Comment:
If I pass `kintsugi install-driver --allow-duplicates` will write the driver command to the gitconfig as:
`kintsugi driver %O %A %B %P --allow-duplicates true` which is incorrect, because `allow-duplcates` doesn't accept any parameter. Either add a parameter to `allow-duplicates` to make it like `interactive-resolution` or solve this some other way.

Reply:
take a look now